src/claude/orchestrator.md

```diff
@@ -462,23 +462,23 @@ Validate:
 3. Test-implementation alignment
 4. Code coverage meets threshold
 
-Return validation verdict: APPROVED or BLOCKED
+Return validation verdict: PASS | FAIL | NEEDS WORK
```

Comment:
@rjmurillo-bot These need to be aligned and documented at `.agents/specs/design/HANDOFF-TERMS.md` with all the other handoff terms between GitHub workflows and agents. A big table of what term means what and how you use them.